crates/store/re_types_core/src/tuid.rs

```diff
@@ -64,8 +64,7 @@ impl Loggable for Tuid {
         ];
         let validity = None;
 
-        // TODO(#3360): We use the extended type here because we rely on it for formatting.
-        Ok(StructArray::new(Self::extended_arrow_datatype(), values, validity).boxed())
+        Ok(StructArray::new(Self::arrow_datatype(), values, validity).boxed())
```

Comment:
@teh-cmc this change here causes the tests to fail against their snapshot with:

```diff
─────────────────────────────────────────────────────────────────────────────────────────────────────────────────
-old snapshot
+new results
────────────┬────────────────────────────────────────────────────────────────────────────────────────────────────
    0     0 │ [
    1     1 │     "1048576 scalars",
    2     2 │     "41.5 MiB in total",
    3       │-    "42 B per row",
          3 │+    "41 B per row",
    4     4 │ ]
```

which sounds like a good thing ™️ and I also think the change still makes sense.
Can you confirm?

Reply:
Are Tuid:s still formatted correctly?

Reply:
<img width="358" alt="image" src="https://github.com/user-attachments/assets/239e4ba1-96ba-44a1-9204-f474ee711489">
this? looks ok

Reply:
via clement in person: don't do this! change it it to a TODO(cmc). Display on rrd dump is affected